drivers/bridge/setup_device.go

```diff
@@ -63,7 +63,7 @@ func setupDefaultSysctl(config *networkConfiguration, i *bridgeInterface) error
 		return nil
 	}
 	if err := ioutil.WriteFile(sysPath, []byte{'0', '\n'}, 0644); err != nil {
-		return fmt.Errorf("libnetwork: Unable to disable IPv6 router advertisement: %v", err)
+		logrus.WithError(err).Info("unable to disable IPv6 router advertisement")
 	}
 	return nil
```

Comment:
This function can no longer return anything except `nil` so you should change the type to no longer return an error.

Reply:
We can't change it because it has to match the `setupStep` type https://github.com/moby/libnetwork/blob/13a4da01ec396d4bd59cc2108a7d183d7ac7fddd/drivers/bridge/setup.go#L3